src/app/extensions/organization-hierarchies/shared/hierarchy-switch/hierarchy-switch.component.spec.ts

```diff
@@ -51,30 +53,28 @@ describe('Hierarchy Switch Component', () => {
     when(organizationHierarchiesFacade.groups$).thenReturn(of([]));
     when(organizationHierarchiesFacade.groupsCount$()).thenReturn(of(0));
     fixture.detectChanges();
-    expect(element.querySelector('select[data-testing-id=hierarchy-switch]')).toBeFalsy();
+    expect(element.querySelector('.hierarchy-switch-current-selection')).toBeFalsy();
   });
 
   it('should be rendered on creation and show options', () => {
     fixture.detectChanges();
-    expect(element.querySelector('select[data-testing-id=hierarchy-switch]')).toBeTruthy();
-    expect(element.querySelector('select[data-testing-id=hierarchy-switch] option[value = root ]')).toBeTruthy();
-    expect(element.querySelector('select[data-testing-id=hierarchy-switch] option[value = root ]').innerHTML).toContain(
-      'ROOT'
-    );
-    expect(
-      element.querySelector('select[data-testing-id=hierarchy-switch] option[value = child ]').innerHTML
-    ).toContain('CHILD');
+    expect(element.querySelectorAll('li')).toHaveLength(2);
+    expect(element.querySelectorAll('li')).toMatchInlineSnapshot(`
```

Comment:
In case you don't want to test the whole html structure you can work with textContent to make the test more robust.